docs/examples/en/loaders/DRACOLoader.html

```diff
@@ -154,6 +154,7 @@ <h2>Source</h2>
 
 		<p>
 			[link:https://github.com/mrdoob/three.js/blob/master/examples/jsm/loaders/DRACOLoader.js examples/jsm/loaders/DRACOLoader.js]
+			[link:https://github.com/mrdoob/three.js/blob/master/examples/js/loaders/DRACOLoader.js examples/js/loaders/DRACOLoader.js]
```

Comment:
This link section only refers to the canonical source files which are the ES6 modules in the JSM directory.